ReactiveXComponent/WebSocket/WebsocketHeader.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ReactiveXComponent.Common;
+
+namespace ReactiveXComponent.WebSocket
+{
+    public class WebSocketHeader
```

Comment:
This class is a duplicate of WebSocketPacket. Should use that class instead and remove this one.